pkg/master/master.go

```diff
@@ -707,9 +712,10 @@ func (m *Master) getExtensionResources(c *Config) map[string]rest.Storage {
 		deploymentStorage := deploymentetcd.NewStorage(restOptions("deployments"))
 		storage["deployments"] = deploymentStorage.Deployment
 		storage["deployments/status"] = deploymentStorage.Status
-		// TODO(madhusudancs): Install scale when Scale group issues are fixed (see issue #18528).
-		// storage["deployments/scale"] = deploymentStorage.Scale
 		storage["deployments/rollback"] = deploymentStorage.Rollback
+		if !m.ApiGroupVersionOverrides["autoscaling/v1"].Disable {
```

Comment:
Did not know the difference between registered vs turned on. Thanks for the clarification. Haven't fixed this yet. Will fix it soon.